sdk/clientcore/annotation-processor/src/main/java/io/clientcore/annotation/processor/utils/ResponseBodyModeGeneration.java

```diff
@@ -57,13 +55,13 @@ public static void generateResponseBodyMode(BlockStmt body) {
     public static void handleResponseBody(BlockStmt body, TypeMirror returnType, java.lang.reflect.Type entityType,
         HttpRequestContext method) {
         body.tryAddImportToParentCompilationUnit(ResponseBodyMode.class);
-        body.tryAddImportToParentCompilationUnit(HttpResponse.class);
-        body.tryAddImportToParentCompilationUnit(HttpResponseAccessHelper.class);
 
+        String typeCast = "Object";
```

Comment:
Can we use entityType instead of the String here ?

Reply:
Only small concern there is `Type.toString()` will be the full `io.clientcore.core.models.binarydata.BinaryData`. For now, I made this default if nothing is found to be `String typeCast = entityType.toString()`.

We can iterate on this further in a future PR.